openapi/api.yaml

```diff
@@ -1360,6 +1362,33 @@ components:
       - $ref: '#/components/schemas/QueryParamsClear'
       - $ref: '#/components/schemas/Datasets'
       - $ref: '#/components/schemas/DataColumnPreview'
+      - properties:
```

Comment:
i think if we add the newly created types to `_generate_schema`, this generation should be a bit cleaner